regionfixer.py

```diff
@@ -573,17 +554,12 @@ def main():
     try:
         freeze_support()
         value = main()
- 
-    except WrongOption as e:
```

Comment:
Is there a reason why you removed this return value?

Reply:
`BadOptionError` comes from the old module `optparse` and `WrongOption` inherits from this class. Also the `NewOptionParser` inherits from the `OptionParser` from the old module `optparse`. Therefore both are no longer used and in my opinion are not necessary.

Reply:
I would really like to keep that return code, could you find a way to add it again using argparse?